modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -37,9 +38,11 @@ This release contains the following changes to connection types:
 * Removed the following connection types:
 ** Username Password
 ** Basic Username Password
-* Added connection pool functionality to improve multi-threading support. With this functionality, each request sent to the API has its own headers and read timeout, which prevents the operation headers from being overridden with the wrong values when operations are executed simultaneously.
+* Added connection pool functionality to improve multi-threading support.
++
+With this functionality, each request sent to the API has its own headers and read timeout, which prevents the operation headers from being overridden with the wrong values when operations are executed simultaneously.
```

Comment:
Each request has its own...headers or header?

Reply:
"headers" is correct